modules/serial.py

```
@@ -0,0 +1,40 @@
+import dataclasses
+
+from akari import (
+    AkariData,
+    AkariDataSet,
```

Ruff (0.8.2)

`akari.AkariDataSet` imported but unused

Remove unused import: `akari.AkariDataSet`

(F401)